### DISCUSSION TITLE
Does the sequence of data set matters when using fparam?

### DISCUSSION THREAD

**Discussion by qdgp** [deepmodeling/deepmd-kit#460] [Q&A] [answered]
I'm using fparam as different temperatures. So do I need to shuffle the raw data set first or will deepmd-kit do it during training? I know that I need to make sure the last set (test set) contains all different temperatures, but I'm not sure if I need to shuffle the rest sets. Thank you for answering my questions.

**Comment by amcadmus** [comment, marked as answer]
DeePMD-kit shuffles the order of frames in the dataset automatically.

**Reply by qdgp** [reply]
Thank you
